core/include/detray/propagator/actors/parameter_resetter.hpp

```diff
@@ -71,7 +71,7 @@ struct parameter_resetter : actor {
         auto& stepping = propagation._stepping;
 
         // Do covariance transport when the track is on surface
-        if (navigation.is_on_module()) {
+        if (navigation.is_on_passive() || navigation.is_on_sensitive()) {
```

Comment:
Just a question: Does this make a difference? ```is_on_module``` is true for every surface that is not a portal, which currently are only passive and sensitive surfaces. I am asking, because ```on_module``` involves less checking